l10n_it_withholding_tax_payment/models/withholding_tax.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2015 Alessandro Camilli (<http://www.openforce.it>)
+# Copyright (c) 2019 Matteo Bilotta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class WithholdingTaxMove(models.Model):
+    _inherit = 'withholding.tax.move'
+
+    wt_move_payment_id = fields.Many2one(
+        'withholding.tax.move.payment', 'Move Payment', readonly=True)
+
+    def unlink(self):
+        for rec in self:
+            if rec.wt_move_payment_id:
+                raise ValidationError(
+                    _(('Warning! Withholding tax move in payment {}: \
+                    you can not delete it').format(
```

Comment:
"you" non dovrebbe essere allineato sotto "Warning"?

Reply:
No, sarebbe un wrong hanging indentation

Reply:
Perfetto, grazie

Reply:
Malissimo! 😂

Ho appena corretto l'indentazione sistemandola un pochino (oltre al fatto che, su questa stringa, era presente il problema degli spazi ad inizio riga).  
In ogni caso, sono stato attento che, la nuova indentazione, passasse dai controlli di `flake8`.